docs/layouts/shortcodes/kairosVersion.html

```diff
@@ -0,0 +1 @@
+{{ "1.6" }}
```

Comment:
I guess this can use json data at this point so we don't need to touch the shortcode each time, no?

Reply:
Yes, it can use $Site.Params. which I now added.